### DISCUSSION TITLE
Version 1.4.0

### DISCUSSION THREAD

**Discussion by gickowtf** [gickowtf/pixoo-homeassistant#16] [Announcements]
**Attention...** this update includes the change to Config Flow. **Please save and delete the entries in your configuration.yaml that are responsible for the Pixoo Divoom 64.** You have to change this config slightly and then adjust it in your newly installed integration. See readme. With this update we are one step closer to being considered an officially recognized HACS integration. With this update I would also like to thank Mrredstone5230 who finally did the programming for the ConfigFlow. Thanks to this update I was able to find the desire again and will finally take care of the new font and my further update plans very soon.

<hr /><em>This discussion was created from the release <a href='https://github.com/gickowtf/pixoo-homeassistant/releases/tag/1.4.0'>Version 1.4.0</a>.</em>

**Comment by SB2DD** [comment]
I mean, you could probably make a PR to the HACS repo right now. It takes a lot of time from their end anyways, so I'm sure by the time they review us it's going to be good.

**Reply by gickowtf** [reply]
Oh yes, that's right! I will do that over the course of the weekend. I also believe that it will go through very well. If everything fits, they are sometimes very fast.

**Comment by gickowtf** [comment]
https://github.com/hacs/default/pull/2397 
Now we have to wait and keep our fingers crossed ;D
